recipes/argparse/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
-  "2.1":
-    url: "https://github.com/p-ranav/argparse/archive/v2.1.tar.gz"
-    sha256: "0a82f464b568b8ee6650fc837f371eb9c81417e2ef9fb3b51f65ad50fa3b8662"
+  "2.2":
```

Comment:
Don't remove older versions, just add the 2.2.

Reply:
I don't think people using cci recipes will care about the include folder layour changing between 2.1 and 2.2.
If somebody's build fails because of this, I guess it's time for him to update the build system as he is now using an outdated argparse release.